### ISSUE TITLE
Build of 3.0.0 snapshot fails day by day

### ISSUE THREAD

**Issue by astesin** [neo4j/neo4j#6715]
Am I a single one who has Neo4j builds failure day by day? My last successful build was
{code}
-rw-rw-r-- 1 stesin stesin 64228516 Mar 12 20:09 neo4j_3.0.0.SNAPSHOT_all.deb
-rw-rw-r-- 1 stesin stesin 72764356 Mar 12 20:09 neo4j-enterprise_3.0.0.SNAPSHOT_all.deb
{code}

Since that. I get

{code}
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:46 h
[INFO] Finished at: 2016-03-15T19:13:21+02:00
[INFO] Final Memory: 116M/1222M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.15:test (default-test) on project neo4j-bolt: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/andrii/workspace/neo4j/community/bolt/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :neo4j-bolt
{code}

What I'm doin wrong?

WBR, Andriii

**Comment by astesin** [comment]
andrii@stesin-notebook:~/workspace/neo4j$ ls -lta /home/andrii/workspace/neo4j/community/bolt/target/surefire-reports | tail -5
-rw-rw-r-- 1 andrii andrii   5112 Mar 15 19:13 TEST-org.neo4j.bolt.security.ssl.TestSslCertificateFactory.xml
-rw-rw-r-- 1 andrii andrii    345 Mar 15 19:13 org.neo4j.bolt.security.ssl.KeyStoreFactoryTest.txt
-rw-rw-r-- 1 andrii andrii   4865 Mar 15 19:13 TEST-org.neo4j.bolt.security.ssl.KeyStoreFactoryTest.xml
-rw-rw-r-- 1 andrii andrii    355 Mar 15 19:13 org.neo4j.bolt.security.auth.BasicAuthenticationTest.txt
-rw-rw-r-- 1 andrii andrii   5658 Mar 15 19:13 TEST-org.neo4j.bolt.security.auth.BasicAuthenticationTest.xml
andrii@stesin-notebook:~/workspace/neo4j$ cat /home/andrii/workspace/neo4j/community/bolt/target/surefire-reports/TEST-org.neo4j.bolt.security.auth.BasicAuthenticationTest.xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.108" tests="9" errors="0" skipped="0" failures="0">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/amd64"/>
    <property name="java.vm.version" value="25.74-b02"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="maven.multiModuleProjectDirectory" value="/home/andrii/workspace/neo4j"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=":"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="fullBuild" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/home/andrii/workspace/neo4j"/>
    <property name="java.runtime.version" value="1.8.0_74-b02"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="line.separator" value="&#10;"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Linux"/>
    <property name="classworlds.conf" value="/usr/share/maven/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="4.2.0-30-generic"/>
    <property name="user.home" value="/home/andrii"/>
    <property name="user.timezone" value="Europe/Kiev"/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="user.name" value="andrii"/>
    <property name="java.class.path" value="/usr/share/maven/boot/plexus-classworlds-2.x.jar"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="/usr/lib/jvm/java-8-oracle/jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher clean install -DfullBuild"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_74"/>
    <property name="java.ext.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/ext:/usr/java/packages/lib/ext"/>
    <property name="securerandom.source" value="file:/dev/./urandom"/>
    <property name="sun.boot.class.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/resources.jar:/usr/lib/jvm/java-8-oracle/jre/lib/rt.jar:/usr/lib/jvm/java-8-oracle/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jsse.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jce.jar:/usr/lib/jvm/java-8-oracle/jre/lib/charsets.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jfr.jar:/usr/lib/jvm/java-8-oracle/jre/classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="/usr/share/maven"/>
    <property name="file.separator" value="/"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.desktop" value="gnome"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase name="shouldBeAbleToUpdateCredentials" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.095"/>
  <testcase name="shouldFailOnUnknownScheme" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.005"/>
  <testcase name="shouldBeAbleToUpdateExpiredCredentials" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0"/>
  <testcase name="shouldFailWhenTooManyAttempts" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.001"/>
  <testcase name="shouldNotBeAbleToUpdateCredentialsIfOldCredentialsAreInvalid" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0"/>
  <testcase name="shouldThrowAndLogOnFailure" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.005"/>
  <testcase name="shouldIndicateThatCredentialsExpired" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0"/>
  <testcase name="shouldNotDoAnythingOnSuccess" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.001"/>
  <testcase name="shouldFailOnMalformedToken" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.001"/>
</testsuite>andrii@stesin-notebook:~/workspace/neo4j$

**Comment by astesin** [comment]
`andrii@stesin-notebook:~/workspace/neo4j$ ls -lta /home/andrii/workspace/neo4j/community/bolt/target/surefire-reports | tail -5
-rw-rw-r-- 1 andrii andrii   5112 Mar 15 19:13 TEST-org.neo4j.bolt.security.ssl.TestSslCertificateFactory.xml
-rw-rw-r-- 1 andrii andrii    345 Mar 15 19:13 org.neo4j.bolt.security.ssl.KeyStoreFactoryTest.txt
-rw-rw-r-- 1 andrii andrii   4865 Mar 15 19:13 TEST-org.neo4j.bolt.security.ssl.KeyStoreFactoryTest.xml
-rw-rw-r-- 1 andrii andrii    355 Mar 15 19:13 org.neo4j.bolt.security.auth.BasicAuthenticationTest.txt
-rw-rw-r-- 1 andrii andrii   5658 Mar 15 19:13 TEST-org.neo4j.bolt.security.auth.BasicAuthenticationTest.xml
andrii@stesin-notebook:~/workspace/neo4j$ cat /home/andrii/workspace/neo4j/community/bolt/target/surefire-reports/TEST-org.neo4j.bolt.security.auth.BasicAuthenticationTest.xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.108" tests="9" errors="0" skipped="0" failures="0">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/amd64"/>
    <property name="java.vm.version" value="25.74-b02"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="maven.multiModuleProjectDirectory" value="/home/andrii/workspace/neo4j"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=":"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="fullBuild" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/home/andrii/workspace/neo4j"/>
    <property name="java.runtime.version" value="1.8.0_74-b02"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="line.separator" value="&#10;"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Linux"/>
    <property name="classworlds.conf" value="/usr/share/maven/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="4.2.0-30-generic"/>
    <property name="user.home" value="/home/andrii"/>
    <property name="user.timezone" value="Europe/Kiev"/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="user.name" value="andrii"/>
    <property name="java.class.path" value="/usr/share/maven/boot/plexus-classworlds-2.x.jar"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="/usr/lib/jvm/java-8-oracle/jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher clean install -DfullBuild"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_74"/>
    <property name="java.ext.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/ext:/usr/java/packages/lib/ext"/>
    <property name="securerandom.source" value="file:/dev/./urandom"/>
    <property name="sun.boot.class.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/resources.jar:/usr/lib/jvm/java-8-oracle/jre/lib/rt.jar:/usr/lib/jvm/java-8-oracle/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jsse.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jce.jar:/usr/lib/jvm/java-8-oracle/jre/lib/charsets.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jfr.jar:/usr/lib/jvm/java-8-oracle/jre/classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="/usr/share/maven"/>
    <property name="file.separator" value="/"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.desktop" value="gnome"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase name="shouldBeAbleToUpdateCredentials" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.095"/>
  <testcase name="shouldFailOnUnknownScheme" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.005"/>
  <testcase name="shouldBeAbleToUpdateExpiredCredentials" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0"/>
  <testcase name="shouldFailWhenTooManyAttempts" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.001"/>
  <testcase name="shouldNotBeAbleToUpdateCredentialsIfOldCredentialsAreInvalid" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0"/>
  <testcase name="shouldThrowAndLogOnFailure" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.005"/>
  <testcase name="shouldIndicateThatCredentialsExpired" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0"/>
  <testcase name="shouldNotDoAnythingOnSuccess" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.001"/>
  <testcase name="shouldFailOnMalformedToken" classname="org.neo4j.bolt.security.auth.BasicAuthenticationTest" time="0.001"/>
</testsuite>andrii@stesin-notebook:~/workspace/neo4j$`

**Comment by jakewins** [comment]
@astesin that last output is rather hard to follow, could you upload your build log (or a least the last few thousand lines of it) to a gist or something? And likewise, can you please include:
- Which OS your are running
- The maven command you are running
- The Java and Maven versions you are running

The builds have been a bit unstable the past several days, but we've shifted new development onto the 3.1 branch and have a team on stabilizing the 3.0 branch, so it should level out in the next week or so. 

Sidenote, If you want stable bleeding-edge builds of Neo4j, you can get fully built ones at http://alpha.neohq.net

**Comment by astesin** [comment]
```
andrii@stesin-notebook:~$ uname -a
Linux stesin-notebook 4.2.0-34-generic #39-Ubuntu SMP Thu Mar 10 22:13:01 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
andrii@stesin-notebook:~$
```

maven command:

```
cd ~/workspace/neo4j/
git pull && \
    time ( date &&  export MAVEN_OPTS="-Xmx2048m" && time mvn clean install -DfullBuild ; date ) > build.OUT 2>&1
```

versions:

```
andrii@stesin-notebook:~/workspace/neo4j$ java -version
java version "1.8.0_74"
Java(TM) SE Runtime Environment (build 1.8.0_74-b02)
Java HotSpot(TM) 64-Bit Server VM (build 25.74-b02, mixed mode)
andrii@stesin-notebook:~/workspace/neo4j$ mvn -version
Apache Maven 3.3.3
Maven home: /usr/share/maven
Java version: 1.8.0_74, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-oracle/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.2.0-34-generic", arch: "amd64", family: "unix"
andrii@stesin-notebook:~/workspace/neo4j$ 

```

**Comment by astesin** [comment]
I'm going to stay with 3.0.\* for a few months.

**Comment by jakewins** [comment]
@astesin cool. That setup should def. work. The error output you posted does not tell us why the tests are failing though, just which tests. I don't remember off the top of my head where the maven build would print the stack traces from those failed tests (since I run the tests from inside my IDE all day..) - but does the `.txt` surefire report contain a stack trace? I have a memory of that giving a more detailed debug output than the `.xml` report you `cat`ed above.

**Comment by astesin** [comment]
Just started yet another attempt to build, I'll tell you whatever is valuable in case of failure.

**Comment by astesin** [comment]
Hmm It looks like I was missing some critical details with git, while development was on the recent move to 3.1 branch; and I'm not a git guru so probably it was my fault somehow. I tried to fix my local repo clone and target it at 3.0 branch, and now I'm looking at what will happen with my build.

Am I correct, that 3.0.0-RELEASE will not be reflected in GitHub repository? How can I track bugfixes and further (sub)releases under 3.0.\* version?

Thanks for your kind patience,
with best regards,
Andrii

**Comment by jakewins** [comment]
We use github branches to track _ongoing_ development - so `git checkout 2.3`, `git checkout 3.0`, `git checkout 3.1` will give you access to the bleeding edge development version of each of those minor versions of Neo4j (remember that we maintain each minor version of Neo4j for several years after the `.0` release goes out). 

To build a specific public release of Neo4j, you'll want to check out a tag of that release. We haven't yet released any stable versions of 3.0, so the closest you'll get is building a milestone release. You can see available tags here: https://github.com/neo4j/neo4j/tags

So, to build the latest milestone of Neo4j 3.0.0, M05, from source, you'd check out the `3.0.0-M05` tag: `git checkout 3.0.0-M05`. That tag will point to the exact commit that we used to test and build the M05 milestone.

**Comment by astesin** [comment]
Dear Jacob, thanks a lot for informing me about the policy. I should have learned git and maven better before posting.

WBR,
Andrii

**Comment by kadamvandita** [comment]
Hello

I am trying to build Neo4j version 3.0 on s390x platform(Red Hat-7.1). The version of java used is 1.8 ,ibm based. Maven version used is 3.0.5.

I am getting lots of test case failures in modules named cypher,cypher-acceptance,Consistency check,Neo4j-enterprise.

According to your post above 3.0 is not stable version.On neo4j website the stable version found was 3.0.Can you please mention to me the stable version released.

if 3.0 is the stable version any idea related to the above mentioned test case failures, would be helpful.

Thanks

**Comment by kadamvandita** [comment]
The maven command used by me was mvn clean install. 
Indivually I ran test cases using command mvn test

**Comment by chrisvest** [comment]
@kadamvandita The `3.0` branch is unstable, but the `3.0.0` tag is stable. When building on Oracle JDK, at least, you have to give Maven more memory before it can complete a build. Also make sure that your check-out is clean, e.g. with `git clean -fdx` before building – in particular you should not have mixed versions of Cypher in your check-out.

**Comment by kadamvandita** [comment]
How do we find out the stable tag on github?

**Comment by chrisvest** [comment]
@kadamvandita We tag every version that we release. There is no "latest" tag.

**Comment by kadamvandita** [comment]
There are test failures in cypher, consistency-check modules .Tested using mvn clean install command

**Comment by kadamvandita** [comment]
HI
I am running neo4j version 2.3 using openjdk 8.I am getting following error 

changing idp max table size should affect IDP inner loop count for Cypher acceptance module.Any solution for the same

**Comment by sinherle** [comment]
Hi,

I am trying to build neo4j 2.3 on Ubuntu 16.04 using openjdk version "1.8.0_91" and maven 3.3.9 on s390x system.
While building cypher acceptance module, there is one test case failing.
Name of test case- **MatchLongPatternAcceptanceTest**
Name of module - Acceptance (path- neo4j/community/cypher/acceptance)

Test case -test("changing idp iteration duration threshold should affect IDP inner loop count") in MatchLongPatternAcceptanceTest.scala is failing with following error-

##  T E S T S

OpenJDK 64-Bit Zero VM warning: ignoring option MaxPermSize=128M; support was removed in 8.0
Picked up _JAVA_OPTIONS: -Xmx8g
Running org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest
Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1,310.577 sec <<< FAILURE! - in org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest
changing idp iteration duration threshold should affect IDP inner loop count(org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest)  Time elapsed: 642.165 sec  <<< ERROR!
org.scalatest.exceptions.TestFailedException: 12 was not less than 12
        at org.scalatest.MatchersHelper$.newTestFailedException(MatchersHelper.scala:160)
        at org.scalatest.Matchers$ShouldMethodHelper$.shouldMatcher(Matchers.scala:6231)
        at org.scalatest.Matchers$AnyShouldWrapper.should(Matchers.scala:6265)
        at org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest$$anonfun$2$$anonfun$apply$mcV$sp$2.apply$mcVI$sp(MatchLongPatternAcceptanceTest.scala:77)
        at org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest$$anonfun$2$$anonfun$apply$mcV$sp$2.apply(MatchLongPatternAcceptanceTest.scala:74)
        at org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest$$anonfun$2$$anonfun$apply$mcV$sp$2.apply(MatchLongPatternAcceptanceTest.scala:74)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest$$anonfun$2.apply$mcV$sp(MatchLongPatternAcceptanceTest.scala:74)
        at org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest$$anonfun$2.apply(MatchLongPatternAcceptanceTest.scala:63)
        at org.neo4j.internal.cypher.acceptance.MatchLongPatternAcceptanceTest$$anonfun$2.apply(MatchLongPatternAcceptanceTest.scala:63)
        at org.scalatest.Transformer$$anonfun$apply$1.apply$mcV$sp(Transformer.scala:22)
        at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
        at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
        at org.scalatest.Transformer.apply(Transformer.scala:22)
        at org.scalatest.Transformer.apply(Transformer.scala:20)
        at org.scalatest.FunSuiteLike$$anon$1.apply(FunSuiteLike.scala:166)
        at org.scalatest.Suite$class.withFixture(Suite.scala:1122)
        at org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite.withFixture(CypherFunSuite.scala:31)
        at org.scalatest.FunSuiteLike$class.invokeWithFixture$1(FunSuiteLike.scala:163)
        at org.scalatest.FunSuiteLike$$anonfun$runTest$1.apply(FunSuiteLike.scala:175)
        at org.scalatest.FunSuiteLike$$anonfun$runTest$1.apply(FunSuiteLike.scala:175)
        at org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
        at org.scalatest.FunSuiteLike$class.runTest(FunSuiteLike.scala:175)
        at org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite.org$scalatest$BeforeAndAfterEach$$super$runTest(CypherFunSuite.scala:31)
        at org.scalatest.BeforeAndAfterEach$class.runTest(BeforeAndAfterEach.scala:255)
        at org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite.runTest(CypherFunSuite.scala:31)
        at org.scalatest.FunSuiteLike$$anonfun$runTests$1.apply(FunSuiteLike.scala:208)
        at org.scalatest.FunSuiteLike$$anonfun$runTests$1.apply(FunSuiteLike.scala:208)
        at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:413)
        at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
        at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:396)
        at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:483)
        at org.scalatest.FunSuiteLike$class.runTests(FunSuiteLike.scala:208)
        at org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite.runTests(CypherFunSuite.scala:31)
        at org.scalatest.Suite$class.run(Suite.scala:1424)
        at org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite.org$scalatest$FunSuiteLike$$super$run(CypherFunSuite.scala:31)
        at org.scalatest.FunSuiteLike$$anonfun$run$1.apply(FunSuiteLike.scala:212)
        at org.scalatest.FunSuiteLike$$anonfun$run$1.apply(FunSuiteLike.scala:212)
        at org.scalatest.SuperEngine.runImpl(Engine.scala:545)
        at org.scalatest.FunSuiteLike$class.run(FunSuiteLike.scala:212)
        at org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite.run(CypherFunSuite.scala:31)
        at org.scalatest.junit.JUnitRunner.run(JUnitRunner.scala:99)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:264)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)

Results :

Tests in error:
  MatchLongPatternAcceptanceTest>CypherFunSuite.run:31->CypherFunSuite.org$scalatest$FunSuiteLike$$super$run:31->CypherFunSuite.runTests:31->CypherFunSuite.runTest:31->CypherFunSuite.org$scalatest$BeforeAndAfterEach$$super$runTest:31->CypherFunSuite.withFixture:31 ? TestFailed

Tests run: 2, Failures: 0, Errors: 1, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 28:24 min
[INFO] Finished at: 2016-06-08T03:08:35-04:00
[INFO] Final Memory: 19M/58M
[INFO] ------------------------------------------------------------------------

Any solution for the same?

**Comment by ketank-new** [comment]
any inputs on @sinherle query .. as eevn i facing the same issue for neo4j v 3.1.0

**Comment by jakewins** [comment]
@ketank-new and @sinherle this ticket is closed, and is not related to problems with MatchLongPatternAcceptanceTest - I guess google finds this ticket since others have asked about that test here. I'm very sorry your question about it wasn't noticed @sinherle.

If you're actively having problem with that test, please consider opening a *new* ticket, and include the following information:

- Your OS and version
- Output from `java -version`
- Output from `git rev-parse HEAD` when ran in your checked out source tree of Neo4j
- Complete maven command used to build Neo4j
- Test failure with stack trace from that latest run (this test has been modified since Neo4j 2.3)

**Comment by ketank-new** [comment]
@jakewins ; posted new issue with details 
https://github.com/neo4j/neo4j/issues/9313
